msteams-platform/concepts/build-and-test/shared-channels.md

```diff
@@ -17,7 +17,8 @@ Microsoft Teams Connect shared channels allow members of a channel to collaborat
 * Individuals and other teams of other organizations.
 
 > [!NOTE]
-> Tab apps in shared channels are available in [Government Community Cloud (GCC), GCC-High, and Department of Defense (DOD)](~/concepts/app-fundamentals-overview.md#government-community-cloud) environments.
+> * Tab apps in shared channels are available in [Government Community Cloud (GCC), GCC-High, and Department of Defense (DOD)](~/concepts/app-fundamentals-overview.md#government-community-cloud) environments.
+> * SharePoint pages and SharePoint apps in shared channels aren't supported in GCC, GCC-High, and DOD environments.
```

Comment:
```suggestion
> * The SharePoint and SharePoint Pages apps aren't currently supported for shared channels in Government Community Cloud (GCC), GCC-High, and Department of Defense (DOD) environments.
```

Reply:
- Branding is SharePoint pages (https://learn.microsoft.com/en-us/sharepoint/dev/solution-guidance/sharepoint-pages-and-the-page-model)

- We don't use the term 'currently' in our doc set

- Abbreviations or acronyms are only expanded in the fist instance of the article. Use the abbreviation/ acronym only for the ret of the article.